sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlanner.java

```diff
@@ -502,4 +510,41 @@ void from(CalcitePlanner planner)
                                          + " to " + this);
     }
   }
+
+  /** Define some tool members and methods for hints. */
+  private static class HintTools
+  {
+    static final HintStrategyTable HINT_STRATEGY_TABLE = createHintStrategies();
+
+    /**
+     * Creates hint strategies.
+     *
+     * @return HintStrategyTable instance
+     */
+    private static HintStrategyTable createHintStrategies()
+    {
+      return HintStrategyTable.builder()
+                              .hintStrategy("broadcast", HintPredicates.JOIN)
+                              .hintStrategy("sort_merge", HintPredicates.JOIN)
+                              //.hintStrategy("sort_merge", HintPredicates.and(HintPredicates.JOIN, joinWithFixedTableName()))
+                              .build();
+    }
+
+    /** Returns a {@link HintPredicate} for join with specified table references. */
+    private static HintPredicate joinWithFixedTableName()
```

Comment:
unused method; comment a few lines above

Reply:
Removed this